src/ResourceManager/Compute/Commands.Compute/Models/PSVirtualMachine.cs

```diff
@@ -33,6 +33,7 @@ public string ResourceGroupName
         {
             get
             {
+                if (string.IsNullOrEmpty(Id)) return null;
```

Comment:
Yes, but this is a fix for a bug I found (Not relavant to this PR, though).  We could update the script later.